angr/analyses/cfg/indirect_jump_resolvers/jumptable.py

```diff
@@ -85,10 +85,8 @@ def resolve(self, cfg, addr, func_addr, block, jumpkind):
         :return: A bool indicating whether the indirect jump is resolved successfully, and a list of resolved targets
         :rtype: tuple
         """
-
         project = self.project  # short-hand
         self._max_targets = cfg._indirect_jump_target_limit
-
         # Perform a backward slicing from the jump target
```

Comment:
Same, do not remove the above two blank lines.

Reply:
ha whoops, i have a feeling breakpoints were here before :D